src/applications/covid-research-volunteer/pages/covidResearchUISchema.js

```diff
@@ -259,11 +266,12 @@ export const uiSchema = {
           On most days, how many people do you have close contact with outside
           of those who live in your home?
         </strong>
+        <br />
+        <br />
+        <strong>Note:</strong> We define close contact as being within 6 feet of
+        a person.
```

Comment:
```suggestion
        <p>
          <strong>Note:</strong> We define close contact as being within 6 feet
          of a person.
        </p>
```
might be better way to achieve desired spacing